<launch>

    <include file="$(find centauro_gazebo)/launch/centauro_world.launch">
        <arg name="realsense" value="true"/>
    </include>

    <node pkg="eurobin_grasp_box" type="xbot2.bash" name="xbot2"/>

    <node pkg="eurobin_grasp_box" type="wait_plugin.py" name="do_homing" args="homing"/>

    <node pkg="eurobin_grasp_box" type="spawn_box.py" name="spawn_box"/>


</launch>